tdrs-frontend/src/components/SplashPage/SplashPage.jsx

```diff
@@ -32,81 +14,94 @@ function SplashPage() {
+              fulfilled their reporting requirements.
+            </p>
+            <button
+              className="usa-button width-full sign-in-button"
+              type="button"
+              size="big"
+              onClick={handleClick}
+            >
+              <div className="mobile:margin-x-auto mobile-lg:margin-0">
+                Sign in with{' '}
+              </div>
+              <img
+                className="mobile:margin-x-auto mobile:padding-top-1 mobile-lg:margin-0 mobile-lg:padding-top-0 width-15 padding-left-1"
+                src={loginLogo}
+                alt="Login.gov"
+              />
+              <span className="visually-hidden">Opens in a new website</span>
+            </button>
+          </div>
+        </div>
       </section>
 
-      <GridContainer className="usa-section padding-right-0">
-        <Grid row gap>
-          <Grid
-            className="resource-info__primary"
-            tablet={{ col: 4 }}
-            mobileLg={{ col: 12 }}
-          >
+      <section className="grid-container usa-section">
+        <div className="grid-row grid-gap">
+          <div className="tablet:grid-col-4">
             <h2 className="resources-header font-heading-2xl margin-top-0 tablet:margin-bottom-0">
               Featured TANF Resources
             </h2>
-            <div className="font-heading-3xs resource-info__secondary">
+            <div className="resource-info__secondary">
               <p>Questions about TANF data?</p>
               <p>
                 Email:{' '}
-                <Link href="mailto: tanfdata@acf.hhs.gov">
+                <a className="usa-link" href="mailto: tanfdata@acf.hhs.gov">
                   tanfdata@acf.hhs.gov
-                </Link>
+                </a>
               </p>
             </div>
-          </Grid>
-          <Grid tablet={{ col: 8 }} mobileLg={{ col: 12 }} row>
-            <CardGroup className="grid-col-12">
-              <Card className="resource-card tablet:grid-col-4 mobile-lg:grid-col-12">
-                <CardHeader>Resource 1</CardHeader>
-              </Card>
-              <Card className="resource-card tablet:grid-col-4 mobile-lg:grid-col-12">
-                <CardHeader>Resource 2</CardHeader>
-              </Card>
-              <Card className="resource-card tablet:grid-col-4 mobile-lg:grid-col-12">
-                <CardHeader>Resource 3</CardHeader>
-              </Card>
-            </CardGroup>
-          </Grid>
-        </Grid>
-      </GridContainer>
-    </div>
+          </div>
+          <div className="tablet:grid-col-8">
+            <ul className="usa-card-group grid-col-12">
+              <li className="usa-card tablet:grid-col-4 mobile-lg:grid-col-12 resource-card">
+                <div className="usa-card__container">
+                  <header className="usa-card__header">
+                    <h2 className="usa-card__heading">Resource 1</h2>
```

Comment:
These `h2` card headings should be `h3`s since they are part of the content that lives underneath/within the `h2` "Featured TANF Resources" heading.

Reply:
Thank you! Fixed